### PR TITLE
File names to camelCase

### LINKED ISSUE #833: Uniform file naming
Modify file names in order to have the same format across the entire project (camelCase). Keep in mind to update imports when doing so

### PR DESCRIPTION
**Pull request by AlejandroLabourdette**
This PR include a lint rule to check file names case
fix #833
> Modify file names in order to have the same format across the entire project (camelCase). Keep in mind to update imports when doing so

### CODE REVIEW

File: src/ast/cairoNodes/export.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export * from './cairoAssert';
 export * from './cairoFunctionDefinition';
 export * from './cairoContract';
-export * from './CairoTempVarStatement';
+export * from './cairoTemVarStatement';
```
Comment (piwonskp) [inline src/ast/cairoNodes/export.ts]:
```suggestion
export * from './cairoTempVarStatement';
```

File: src/ast/cairoNodes/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export * from './cairoAssert';
 export * from './cairoContract';
 export * from './cairoFunctionDefinition';
-export * from './CairoTempVarStatement';
+export * from './cairoTemVarStatement';
```
Comment (piwonskp) [inline src/ast/cairoNodes/index.ts]: Typo in the name of the file

File: src/passes/builtinHandler/export.ts [hunk 3]
```diff
@@ -4,5 +4,5 @@ export * from './explicitConversionToFunc';
 export * from './msgSender';
 export * from './thisKeyword';
 export * from './ecrecover';
-export * from './MathsOperationToFunction';
+export * from './mathsOperationToFunctionBh';
```
Comment (piwonskp) [inline src/passes/builtinHandler/export.ts]: Bh?

File: tests/interface_call_forwarder/interface_forwarder.test.ts [hunk 4]
```diff
@@ -1,295 +0,0 @@
-import { expect } from 'chai';
```
Comment (piwonskp) [inline tests/interface_call_forwarder/interface_forwarder.test.ts]: Why did you remove the file?

Comment (AlejandroLabourdette) [reply tests/interface_call_forwarder/interface_forwarder.test.ts]: That's odd

Comment (AlejandroLabourdette) [reply tests/interface_call_forwarder/interface_forwarder.test.ts]: Problem was with this [logic](https://github.com/NethermindEth/warp/blob/97e70b0f48114e219f6ba9dfbf4947b3d1574950/tests/interfaceCallForwarder/interfaceForwarder.test.ts#L288) to delete generated files in the folder
Filename wasn't updated

**Review by rjnrohit — APPROVED** [review]

File: src/utils/nodeTypeProcessing.ts [hunk 5]
```diff
@@ -38,6 +38,7 @@ import { TranspileFailedError } from './errors';
 import { error } from './formatting';
 import { getContainingSourceUnit } from './utils';
 import { getNodeType, getNodeTypeInCtx } from './typeStrings/typeStringParser';
+import { CairoAssert } from '../ast/cairoNodes'; // eslint-disable-line
```
Comment (piwonskp) [inline src/utils/nodeTypeProcessing.ts:41]: Why eslint-disable-line is needed?

**Review by piwonskp — APPROVED** [review]

**Comment by swapnilraj** [comment]
Hey we only need the src files to be camel case, the files in example_contracts can be ignored by this rule

File: package-lock.json [hunk 6]
```diff
@@ -0,0 +1,16086 @@
+{
+  "name": "@nethermindeth/warp",
```
Comment (swapnilraj) [inline package-lock.json]: can you delte this?
This is a npm artifact

**Review by swapnilraj — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 6 above)
Comment (AlejandroLabourdette) [reply package-lock.json:2]: you mean the file?

Comment (AlejandroLabourdette) [reply package-lock.json]: you mean the file?

**Comment by AlejandroLabourdette** [comment]
> Hey we only need the src files to be camel case, the files in example_contracts can be ignored by this rule

Hi, lint just inspect .ts files by default, Solidity files are not examined, but I change them anyway to have the same naming pattern.

**Comment by JorikSchellekens** [comment]
Can you make the lint work for js files as well?

**Comment by AlejandroLabourdette** [comment]
> Can you make the lint work for js files as well?

It also works for js, the above idea was that sol files could not be parsed by eslint.

**Review by swapnilraj — APPROVED** [review]
